src/charm.py

```diff
@@ -63,7 +64,7 @@ class AlertmanagerCharm(CharmBase):
     # Container name is automatically determined from charm name
     # Layer name is used for the layer label argument in container.add_layer
     # Service name matches charm name for consistency
-    _container_name = _layer_name = _service_name = "alertmanager"
+    _container_name = _layer_name = _service_name = _exe_name = "alertmanager"
```

Comment:
Why `exe` and not `bin`? Perhaps we talked too long about WSL :D
(Jk, this is _not_ a change request)

Reply:
I was going to do _executable_name then decided it should be shorter 😁